vmdb/app/models/miq_action.rb

```diff
@@ -404,7 +404,6 @@ def action_check_compliance(action, rec, inputs)
   end
 
   def action_tag(action, rec, inputs)
-    get_policies_from = inputs[:get_policies_from]
```

Comment:
action_tag became action_tag_inherit here:
```
1c96d3c6847d6ef1cbd2e2a8b6a96d29bb40af4b
Date:   Wed May 21 20:27:01 2008 +0000

    BugzID: 2327, 2534
    Renamed original tag action from "tag" to "tag_inherit"
    Added new action "tag" that will tag the target object with the specified management tag.
```

When action_tag was duplicated as action_tag_inherit, the new action_tag method no longer logged the `get_policies_from` but never removed the unused variable assignment.